include/caffe/util/nccl.hpp

```diff
@@ -0,0 +1,77 @@
+#ifndef CAFFE_UTIL_NCCL_H_
+#define CAFFE_UTIL_NCCL_H_
+#ifdef USE_NCCL
+
+#include <nccl.h>
+
+#include "caffe/common.hpp"
+
+#define NCCL_CHECK(condition) \
+  { \
+    ncclResult_t result = condition; \
+    CHECK_EQ(result, ncclSuccess) << " " \
+      << ncclGetErrorString(result); \
+  }
+
+#if 0
+inline const char* ncclGetErrorString(ncclResult_t result) {
```

Comment:
No, we don't need that - I can't remember why it was there, but it certainly shouldn't be anymore :)

> On Jun 1, 2016, at 8:20 PM, Sergei Nikolaev <notifications@github.com> wrote:
> 
> In include/caffe/util/nccl.hpp:
> 
> > +#define CAFFE_UTIL_NCCL_H_
> > +#ifdef USE_NCCL
> > +
> > +#include <nccl.h>
> > +
> > +#include "caffe/common.hpp"
> > +
> > +#define NCCL_CHECK(condition) \
> > +  { \
> > +    ncclResult_t result = condition; \
> > +    CHECK_EQ(result, ncclSuccess) << " " \
> > +      << ncclGetErrorString(result); \
> > +  }
> > +
> > +#if 0
> > +inline const char* ncclGetErrorString(ncclResult_t result) {
> @slayton58 do we need it?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>